nifi-nar-bundles/nifi-box-bundle/nifi-box-services-nar/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add LICENSE and NOTICE files to nifi-box-services-nar and nifi-box-api-services-nar.